packages/common/src/components/TableView/ManageColumnsToolbar.tsx

```diff
@@ -41,23 +41,24 @@ export const ManageColumnsToolbar = ({
   resourceFields,
   setColumns,
   defaultColumns,
+  labels = defaultLabels,
 }: ManageColumnsToolbarProps) => {
-  const { t } = useTranslation();
   const [isOpen, setIsOpen] = useState(false);
   return (
     <ToolbarItem>
-      <Tooltip content={t('Manage Columns')}>
-        <Button variant="plain" onClick={() => setIsOpen(true)} aria-label={t('Manage Columns')}>
+      <Tooltip content={labels.manageColumns}>
+        <Button variant="plain" onClick={() => setIsOpen(true)} aria-label={labels.manageColumns}>
           <ColumnsIcon />
         </Button>
       </Tooltip>
       <ManageColumns
         showModal={isOpen}
         onClose={() => setIsOpen(false)}
-        description={t('Selected resourceFields will be displayed in the table.')}
+        description={labels.description}
```

Comment:
hmm, labels are used for the fields ... doesn't look like it should be overloaded for a generic title ...
adding the title as a separate prop can be a path for future refactor

Reply:
`labels` prop is not used for the fields - it's a dictionary of all labels used by the dialog